src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,131 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../Types/Todo';
+import usersFromServer from '../../api/users';
+import { getUserById } from '../../services/user';
+
+type Props = {
+  onSubmit: (toDo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ onSubmit }) => {
+  const [userId, setUserId] = useState<number>(0);
+  const [title, setTitle] = useState<string>('');
+  const [hasTitleError, setHasTitleError] = useState<boolean>(false);
+  const [hasUserError, setHasUserError] = useState<boolean>(false);
+
+  const reset = () => {
+    setTitle('');
```

Comment:
You can use a [Group Related State](https://react.dev/learn/choosing-the-state-structure#group-related-state), so when you reset your code should look like

```js
const reset = () => setForm(initialState)
```